src/devices/UFireIse/UFirePh .cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire
+{
+    /// <summary>
+    /// Get pH measuremens from μFire ISE (ion-selective electrode) Probe Interface
+    /// </summary>
+    public class UFirePh : UFireIse
+    {
+        private const float ProbeMvToPh = 59.2F;
+        private const float TemperatureCorrectionFactor = 0.03F;
+
+        /// <summary>
+        /// pH (Power of Hydrogen) units measurement
+        /// </summary>
+        public float Ph = 0;
```

Comment:
I assume UnitsNet doens't have unit for that? Acidity or something?

Reply:
yes UnitsNet do not have a unit for pH (Power of Hydrogen)